src/components/Card/Card.tsx

```diff
@@ -0,0 +1,37 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core"
+import { PropsOf } from "../../utils/types"
+import { ThemeCss } from "../../theme"
+
+const baseCss: ThemeCss = theme => ({
+  background: theme.colors.white,
+})
+
+const standaloneCss: ThemeCss = theme => ({
+  paddingTop: theme.space[6],
+  paddingBottom: theme.space[5],
+  paddingLeft: theme.space[7],
+  paddingRight: theme.space[3],
```

Comment:
```suggestion
  paddingRight: theme.space[7],
```

I might be missing something here! 🤔

Reply:
I actually got these styles from Build cards, which again demonstrates that we're are not very consistent 😆 

Perhaps we should have card variants which would define what paddings to use? I don't expect that we are going to always be able to maintain the same padding, but having a variant should at least restrict us a little bit :)

Reply:
Yep,  defs 👍 for variants. Will try to boil things down!
… and again, resolving!